docs/readmes/know_issues.md

```diff
@@ -0,0 +1,11 @@
+## aad-pod-identity - Known Issues
```

Comment:
Let's add this docs to either https://github.com/Azure/aad-pod-identity/blob/master/README.md or https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.md